dd-smoke-tests/play-2.7/application/project/plugins.sbt

```
@@ -0,0 +1,2 @@
+val playVersion = System.getProperty("play.version", "2.8.2")
```

```suggestion
val playVersion = System.getProperty("play.version", "2.7.5")
```

(Assuming `playVersion` is correct.)

---

Thanks